src/Microsoft.Restier.WebApi/Formatter/Deserialization/DefaultRestierDeserializerProvider.cs

```diff
@@ -11,11 +11,24 @@
 namespace Microsoft.Restier.WebApi.Formatter.Deserialization
 {
     /// <summary>
-    /// The default serializer provider.
+    /// The default deserializer provider.
     /// </summary>
-    internal class DefaultRestierDeserializerProvider : DefaultODataDeserializerProvider
+    public class DefaultRestierDeserializerProvider : DefaultODataDeserializerProvider
     {
         private RestierEnumDeserializer enumDeserializer;
+        private static readonly Lazy<DefaultRestierDeserializerProvider> lazy
```

Comment:
I mean if DefaultRestierDeserializerProvider is thread safe? This static instance could be called at the same time by multiple request?